### ISSUE TITLE
Question about linearized NNs

### ISSUE THREAD

**Issue by bwnjnOEI** [gortizji/linearized-networks#1]
Hi, nice work! You expanded and delved the work [Deep learning versus kernel learning: an empirical study of loss landscape geometry and the time evolution of the Neural Tangent Kernel](https://arxiv.org/abs/2010.15110). But some problem puzzle me:

What is `the linearized training`? This seem like different from training NNs using tiny learning rate(gradient flow). And i guess it just update parameters <img src="https://render.githubusercontent.com/render/math?math=\theta \quad \text{  in  } f^{lin}_{\theta}(x) = f_{\theta_0}(x) %2B \nabla f_{\theta_0}(x)(\theta - \theta_0)"> by SGD( or using tiny learning rate)?

The question I have is not directly from your paper, but I want to solve my trouble.

**Comment by gortizji** [comment]
Linearized training refers indeed to updating the parameters of the linearized model (i.e., first-order Taylor decomposition). This is in general not equivalent to training the full neural network with gradient flow, which would also update higher order terms in the decomposition.
